walkthroughs/week-3-libraries/charts/examples/favorite-colors/src/main/webapp/script.js

```diff
@@ -0,0 +1,39 @@
+// Copyright 2019 Google LLC
```

Comment:
I'm assuming the examples (like bigfoot, favorite colors) are unmodified by you.

Reply:
Yup, didn't modify any examples